rust/actyx/node-manager-bindings/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-edition = { workspace = true }
+edition = "2021"
```

Comment:
I don't understand why we're changing this one. If we set a version on the workspace we get to move everything along at the same time and force a more cohesive codebase. What am I missing?

Reply:
I had started switching this to workspace settings, but it is now two years later and I still get errors about this in VS Code, so I reverted it. Now that we have much fewer crates, there is less reason to simplify the update in 2024.